label-application-rule-config-service-impl/src/main/java/org/hypertrace/label/application/rule/config/service/LabelApplicationRuleValidatorImpl.java

```diff
@@ -25,6 +31,8 @@
 import org.hypertrace.label.application.rule.config.service.v1.UpdateLabelApplicationRuleRequest;
 
 public class LabelApplicationRuleValidatorImpl implements LabelApplicationRuleValidator {
+  private static final String STRING_SEPARATOR = ",";
```

Comment:
label rule should not be parsing a comma separated list - we had discussed accepting either a string or list of strings.